crates/typst-library/src/foundations/decimal.rs

```diff
@@ -435,13 +436,15 @@ pub enum ToDecimal {
     Int(i64),
     /// A float to be converted to the equivalent decimal.
     Float(f64),
+    Decimal(Decimal),
```

Comment:
could reorder this enum and the `cast!` to be consistent with `ToInt` and `ToFloat`? (and add a short doc comment just for consistency)

the ordering affects the error messages, so it'd be nice if everything would be consistent.

Reply:
Done. Should the cast also accept bools like `ToInt` and `ToFloat`?

Reply:
While I find it odd to begin with, it does seem that it would be consistent indeed.

Reply:
For int at least it makes sense I think. And the rest is consistency. In Rust maybe more since it's a systems language but 1 for true and 0 for false are kind of universal.